src/main/java/homework2/BaseID.java

```diff
@@ -0,0 +1,6 @@
+package homework2;
+
+public interface BaseID <T> {
```

Comment:
Could be Abstract clas.. If yes, we can add getter and setter here with the realization. And in the task we have numeric string

Reply:
so we will change ID for ticket to int?

Reply:
id, long.. up to you))

Reply:
Users also need id?

Reply:
you can add it)